tests/conftest.py

```diff
@@ -493,6 +494,30 @@ def assert_resp_response(r, response, resp2_expected, resp3_expected):
         assert response == resp3_expected
 
 
+def assert_geo_is_close(coords, expected_coords):
+    """
+    Verifies that the coordinates are close within the floating point tolerance
+
+    Valkey uses 52-bit presicion
+    """
+    check_geo = list(zip(coords, expected_coords))
+
+    for i in check_geo:
+        assert math.isclose(*i)
```

Comment:
There is no need to force the conversion to list here.
You could do something like:
```suggestion
    for a, b in zip(coords, expected_coords):
        assert math.isclose(a, b)
```